core/common/src/main/java/alluxio/wire/GetConfigurationOptions.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.wire;
+
+import alluxio.thrift.GetConfigurationTOptions;
+
+import com.google.common.base.Objects;
+
+/**
+ * Options for backing up the Alluxio master.
+ */
+public class GetConfigurationOptions {
+  private boolean mRawValue;
```

Comment:
final?

Reply:
added a setter. so final is no more valid